src/test/spec/unified_runner/operation.rs

```diff
@@ -285,6 +287,8 @@ pub(super) struct DeleteOne {
     session: Option<String>,
     #[serde(flatten)]
     options: DeleteOptions,
+    #[serde(rename = "comment")]
+    _comment: Option<Bson>,
```

Comment:
Can we add TODO comments to this field (and the other `_comment` fields below) to delete these workarounds when RUST-1071 is done? You can just copy the one you deleted from `CountDocuments`.